po/es.po

```diff
@@ -788,22 +788,22 @@ msgstr ""
 
 #: ../build/src/preferences_gen.h:3421
 msgid "demosaicing for zoomed out darkroom mode"
-msgstr "La interpolación cromática para alejar en el modo de cuarto oscuro"
+msgstr "Interpolación cromática usada para ampliar en el modo de cuarto oscuro"
 
 #: ../build/src/preferences_gen.h:3432
 msgctxt "preferences"
 msgid "always bilinear (fast)"
-msgstr "siempre bilineal (rápido)"
+msgstr "Siempre bilineal (rápido)"
 
 #: ../build/src/preferences_gen.h:3437 ../build/src/preferences_gen.h:3457
 msgctxt "preferences"
 msgid "at most PPG (reasonable)"
-msgstr "la mayoría de PPG (razonable)"
+msgstr "Como máximo PPG (aceptable)"
 
 #: ../build/src/preferences_gen.h:3442
 msgctxt "preferences"
 msgid "full (possibly slow)"
-msgstr "completo (posiblemente lento)"
+msgstr "Completo (posiblemente lento)"
```

Comment:
That would be correct if "Completa" should refer to interpolation. But an interpolation cannot be incomplete. "Full" tampoco es un tipo de interpolación sino una combinación de varios especificados, por lo que se refiere al modo (palabra implícita) completo (están los modos rápido y aceptable o intermedio). It helps a lot the tooltips to understand. But I will add "modo" to avoid ambiguities. Thanks.